### DISCUSSION TITLE
Fix behavior for `less` program

### DISCUSSION THREAD

**Discussion by TylerTemp** [Eugeny/tabby#6946] [Q&A]
related: <https://github.com/Eugeny/tabby/discussions/6936>

Hi,

I'm expecting that commands like `less` can behave like under Linux:
mouse wheel up & down will let `less` scroll the content. Currently, I can `alt + scroll`, but the scroll up will pass outside the top of the `less` program.

What I expected:
1. No `alt` needed for `less` (and `more`) works properly when scrolling the mouse wheel
2. scroll will not bypass the `less` window itself

Any suggestions are welcome! I've googled and searched Github issue but found no result.

Thanks!

**Comment by jkuijkuihahahehe** [comment]
的确 less 用方向键上行滚动的时候 会出现错误   感觉都不想用tabby了

**Reply by jkuijkuihahahehe** [reply]
最新版本貌似修复了 现在less可以正常使用
